src/components/attendee/VideoHuddle/VideoHuddle.tsx

```diff
@@ -72,41 +50,25 @@ const ActiveVideoHuddle: React.FC<ActiveVideoHuddleProps> = ({
         </a>
       </div>
       <div className={styles.VideoHuddleVideos}>
+        {/*  TODO shouldMirrorVideo */}
         {localParticipant && (
-          <div className={styles.VideoContainer} key={localParticipant.sid}>
-            <VideoParticipant
+          <div className={styles.VideoContainer} key={localParticipant.id}>
+            <VideoCommsParticipant
+              isLocal={true}
               participant={localParticipant}
-              shouldMirrorVideo
+              extraButtons={extraButtons}
```

Comment:
Suggested above, instead of just buttons, if `VideoCommsParticipant`...
(how about `Vidcom` as a system? Sounds weird? O.o) 
... takes in children, we can be flexible at what kind of buttons/icons/components we can add